Service/src/routes/charts.py

```diff
@@ -43,6 +52,7 @@ def get_history(ticker, length):
 
    try:
       response = rawResponse.json()
+      g.log.info(response, isPprint=True)
```

Comment:
this is a fat log that shouldn't be printed
blocker for merge

Reply:
Removed